lmp/bb-config.sh

```diff
@@ -135,14 +135,16 @@ EOFEOF
 sed -e "s|@@ARCHIVE@@|${archive}|" ${HERE}/bb_logconfig.json > bb_logconfig.json
 
 # Additional packages based on the CI job used
-if [ "$CONF_VERSION" == "1" ]; then
-	cat << EOFEOF >> conf/local.conf
+if [ -n ${EXTRA_IMAGE_INSTALL} ];then
```

Comment:
Add "" before/after ${EXTRA_IMAGE_INSTALL} to make sure it is always valid. Please also add space after ; and before then.

Reply:
Thanks but don't get the part to add the space before
```
if [ -n "${EXTRA_IMAGE_INSTALL}" ] ; then
```
is this what you mean?

Reply:
Sorry, just before then is enough :-)

`if [ -n "${EXTRA_IMAGE_INSTALL}" ]; then
`

Reply:
fixed